src/frontend/components/dashboard/MapTab/mapboxStyle.js

```diff
@@ -18,7 +18,8 @@ const mbStyle = {
     },
     composite: {
       type: 'vector',
-      url: 'mapbox://jeremiak.4hjgevzj', //'mapbox://jeremiak.27hyzjbx',
+      // tiles: ['http://localhost:8080/data/piecewise/{z}/{x}/{y}.pbf'],
+      url: 'mapbox://jeremiak.apm7cltf',
```

Comment:
This will change to a tileset that's hosted on Google Cloud Storage but for now its on Mapbox to make reviewing easier